bindings/pyroot/ROOT.py

```diff
@@ -241,23 +241,10 @@ def _TTree__iter__( self ):
 _root.CreateScopeProxy( "TTree" ).__iter__    = _TTree__iter__
 
 # Array interface
-def _proxy__array_interface__(self):
-    getter_array_interface = "_get__array_interface__"
-    if hasattr(self, getter_array_interface):
-        return self._get__array_interface__()
-    else:
-        raise Exception("Class {} does not have method {}.".format(
-            type(self), getter_array_interface))
-
-for pyclass in [
-        "std::vector<{dtype}>",
-        "ROOT::VecOps::RVec<{dtype}>"
-        ]:
-    dtypes = ["float", "double", "int", "unsigned int", "long", "unsigned long"]
-    for dtype in dtypes:
-        class_scope = _root.CreateScopeProxy(pyclass.format(dtype=dtype))
-        class_scope._proxy__array_interface__ = _proxy__array_interface__
-        class_scope.__array_interface__ = property(class_scope._proxy__array_interface__)
+def _add__array_interface__(self):
```

Comment:
The only "ugly" thing is that we have to inject a method in the ROOT module.

Reply:
We could also implement `_add__array_interface__` in a helpers.py and get a pointer to that method in `Pythonize.cxx`, right? It is not super important anyway since in the new PyROOT we will not have this problem.

Reply:
We would need to add the `helper.py` as a part of the ROOT module. This seems more odd to me than putting it in `ROOT.py` directly. However, you're right. The new pyroot does solve this naturally much nicer.